make_sys.sh

```diff
@@ -18,15 +18,13 @@ DEFINES=("-DTRACY_ENABLE" "-DTRACY_MANUAL_LIFETIME" "-DTRACY_FIBERS")
 
 echo "::set-output name=tracy-tag::$TAG"
 mkdir -p "$DESTINATION"
-
 curl -sL "$TARBALL" -o - | tar -f - -zxC "$DESTINATION"
-
 BASEDIR=("$DESTINATION"/*)
-REQUIRED=($(gcc --dependencies ${DEFINES[@]} "$BASEDIR/TracyClient.cpp" | grep -o "$BASEDIR/[^ \\]*"))
 
-mkdir -p "tracy-client-sys/tracy"
+cp -r "$BASEDIR/public/"* "tracy-client-sys/tracy/"
+cp "$BASEDIR/LICENSE" "tracy-client-sys/tracy/"
 
-bindgen "$BASEDIR/TracyC.h" \
+bindgen "tracy-client-sys/tracy/TracyC.h" \
```

Comment:
I think the issue is somewhere here, there are two `TracyC.h` files in the tree, only one of them has been updated and bindgen uses the old one, that's my theory anyway.

Reply:
Great catch. I did notice the double file while investigating the SIGSEGV and made a note in my head to investigate it, only to promptly forget all about it. Thanks for writing it down!